packages/tools/pie-wrapper-react/CHANGELOG.md

```diff
@@ -4,7 +4,7 @@
 
 ### Minor Changes
 
-- [Changed] - `@lit-labs/react` dependency to `@lit/react` and updated any references. ([#999](https://github.com/justeattakeaway/pie/pull/999)) by [@siggerzz](https://github.com/siggerzz)
+- [Changed] - `@lit/react` dependency to `@lit/react` and updated any references. ([#999](https://github.com/justeattakeaway/pie/pull/999)) by [@siggerzz](https://github.com/siggerzz)
```

Comment:
I believe this one should still be `@lit-labs/react`

Reply:
fixed 💯